internal/datanode/flow_graph_manager.go

```diff
@@ -137,11 +137,13 @@ func (fm *flowgraphManager) addAndStart(dn *DataNode, vchan *datapb.VchannelInfo
 }
 
 func (fm *flowgraphManager) release(vchanName string) {
-	if fg, loaded := fm.flowgraphs.GetAndRemove(vchanName); loaded {
+	if fg, loaded := fm.flowgraphs.Get(vchanName); loaded {
```

Comment:
`Get` then `Remove` may cause double `fg.close()`, is it safe?

Reply:
yes, its sync.Once close